### PR TITLE
people: updated @NeoChen1024

### PR DESCRIPTION
**Pull request by NeoChen1024**

### CODE REVIEW

File: data/people.yml [hunk 1]
```diff
@@ -166,11 +166,16 @@
     package updates, and various ports.
   rurl:
 -
-  fullname: Neo Chen
+  fullname: Kelei Chen
   username: NeoChen1024
-  description:
-  longdesc:
-  rurl:
+  description: Self-claimed computer historian, the creator of NeoNetwork
```
Comment (MingcongBai) [inline data/people.yml:171]: Self-proclaimed computer historian, creator (founder) of NeoNetwork

File: data/people.yml [hunk 2]
```diff
@@ -166,11 +166,16 @@
...
+  longdesc: >
+    Interested in computers from 1940's to 1980's, including supercomputers, mainframes, to
```
Comment (MingcongBai) [inline data/people.yml:173]: Two choices here...

- Irom ..., ..., to ...
- Including ..., ..., and ...

File: data/people.yml [hunk 3]
```diff
@@ -166,11 +166,16 @@
...
+    microcontrollers. Have difficulties dealing with modern software development and OOP languages.
```
Comment (MingcongBai) [inline data/people.yml:174]: Intolerant to modern software...?

File: data/people.yml [hunk 4]
```diff
@@ -166,11 +166,16 @@
...
+    Sort of a amateur radio enthusiast.
```
Comment (MingcongBai) [inline data/people.yml:175]: "an amateur"

File: data/people.yml [hunk 5]
```diff
@@ -166,11 +166,16 @@
...
+    Plays Girls Frontline (TW Server).
```
Comment (MingcongBai) [inline data/people.yml:176]: Girl's Frontline *

Comment (liushuyu) [reply data/people.yml:176]: _Girls' Frontline_

File: data/people.yml (hunk 2 above)
Comment (liushuyu) [reply data/people.yml:173]: > from 1940's to 1980's

from the 1940s to 1980s?

**Comment by MingcongBai** [comment]
Okay! Welcome to the gang!

**Pull request merged**